quantum/led_matrix/led_matrix.c

```diff
@@ -635,3 +669,31 @@ void led_matrix_set_flags(led_flags_t flags) {
 void led_matrix_set_flags_noeeprom(led_flags_t flags) {
     led_matrix_set_flags_eeprom_helper(flags, false);
 }
+
+#if LED_DISABLE_TIMEOUT > 0
+uint32_t led_matrix_disable_timeout_get(void) {
+    return led_disable_timeout;
+}
+
+void led_matrix_disable_timeout_set(uint32_t timeout) {
+    led_disable_timeout = timeout;
+}
+void led_matrix_disable_time_reset(void) {
+    led_anykey_timer = 0;
+}
+#endif // LED_DISABLE_TIMEOUT > 0
+
+#ifdef LED_MATRIX_DRIVER_SHUTDOWN_ENABLE
+void led_matrix_driver_shutdown(void) {
+    led_matrix_driver.shutdown();
```

Comment:
This isnt safe, as the other backend drivers do not implement the function. They should be explicitly initialled to NULL and  `led_matrix_driver.shutdown`  checked before use